ciao-launcher/main.go

```diff
@@ -538,6 +582,12 @@ func main() {
 		log.Fatalf("Unable to initialise logs: %v", err)
 	}
 
+	if configError != nil {
+		log.Fatalf("Failure in Config files: %v", configError)
```

Comment:
Not sure we want this one to be fatal. If the right information was provided through the command line, a wrongly formatted configuration file should only be a warning and not prevent us from starting launcher.